src/components/UpdateEmailCheckEmail.vue

```diff
@@ -41,7 +41,7 @@ onBeforeMount(async () => {
   </p>
   <p>
     Didn't receive the email? Check your spam filter for an email from
-    account@climateiq.org.
+    noreply@climateiq.org.
```

Comment:
noreply.climateiq.firebaseapp.com

Reply:
It's supposed to be norely@climateiq.org - we're working on getting the email domain validated right now.